### ISSUE TITLE
Matthews Correlation Coefficient - Classification Metric

### ISSUE THREAD

**Issue by nkumar115** [nkumar115/Data-Science#7]
TL;DR
scalar metric for binary classification: the Matthews Correlation Coefficient (MCC)

**Link**

https://towardsdatascience.com/the-best-classification-metric-youve-never-heard-of-the-matthews-correlation-coefficient-3bf50a2f3e9a

**Takeaway**

The higher the correlation between true and predicted values, the better the prediction. This is the phi-coefficient (φ), rechristened Matthews Correlation Coefficient (MCC) when applied to classifiers.

MCC takes into account all four values in the confusion matrix, and a high value (close to 1) means that both classes are predicted well, even if one class is disproportionately under- (or over-) represented.

**Screenshot**

**Tool** 

Python
